setup.cfg

```diff
@@ -1,4 +1,6 @@
 [flake8]
-exclude = validation
+docstring-convention = numpy
+format = pylint
+ignore = E203 W503 C901
```

Comment:
According to the [documentation](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-ignore), I think the list should be comma separated and not space separated.
And instead of `ignore`, maybe we could switch to `extend-ignore` since it's the value specified in the [configuration documentation](https://flake8.pycqa.org/en/latest/user/configuration.html).